adoc/deployment-sysreqs.adoc

```diff
@@ -203,6 +203,25 @@ The management workstation needs at least the following networking permissions:
 
 |===
 
+==== Port Forwarding
+
+The link:https://kubernetes.io/docs/concepts/cluster-administration/networking/[{kube} networking model] requires that your nodes have IP forwarding enabled in the kernel.
+`skuba` checks this value when installing your cluster and installs a rule in `/etc/sysctl.d/90-skuba-net-ipv4-ip-forward.conf` to make it persistent.
+
+Other software can potentially install rules with higher priority overriding this value and causing the cluster to fail.
+
+You can manually check if this is enabled using the following command:
+
+[source,bash]
+----
+# sysctl net.ipv4.ip_forward
+
+net.ipv4.ip_forward = 1
+----
+
+`net.ipv4.ip_forward` must be set to `1`.
+
+
```

Comment:
Additionally, you can check in what order persisted rules are processed by running `sysctl --system -a`.